front/app/containers/Admin/dashboard/components/TimeControl.tsx

```diff
@@ -15,10 +15,11 @@ import messages from '../messages';
 import styled from 'styled-components';
 import { colors } from 'utils/styleUtils';
 
-const Container = styled.div`
+const Container = styled.div<{ width?: string }>`
```

Comment:
I think this could be a `Box`, I think all of the css properties here are on that component

Reply:
True, but then I would have to inject the theme and use that for the border radius, which adds some boilerplate too. But I will remove the unused `width` prop here